bitnami/mediawiki/1/debian-12/rootfs/opt/bitnami/scripts/mediawiki-env.sh

```diff
@@ -81,6 +82,7 @@ export MEDIAWIKI_DATA_TO_PERSIST="${MEDIAWIKI_DATA_TO_PERSIST:-images extensions
 export MEDIAWIKI_SKIP_BOOTSTRAP="${MEDIAWIKI_SKIP_BOOTSTRAP:-}" # only used during the first initialization
 export MEDIAWIKI_WIKI_NAME="${MEDIAWIKI_WIKI_NAME:-Bitnami MediaWiki}" # only used during the first initialization
 export MEDIAWIKI_WIKI_PREFIX="${MEDIAWIKI_WIKI_PREFIX:-/wiki}" # only used during the first initialization
+export MEDIAWIKI_SCRIPT_PATH="${MEDIAWIKI_SCRIPT_PATH:-/w}" # only used during the first initialization
```

Comment:
Why the default value `MEDIAWIKI_SCRIPT_PATH=/w`?

If possible, I would recommend this feature be an enhancement to available settings instead of modifying the current default feature.

Reply:
that was to match the upstream default: https://www.mediawiki.org/wiki/Manual:$wgScriptPath

Reply:
I'm sorry but I think I can not accept this PR unless the default change is reversed to an empty string, as it may cause issues with existing MediaWiki deployments because of the following would happen:

- Setting `$wgScriptPath` is only set during the first initialization, therefore it won't be updated for existing deployments.
- Because we moved the webserver configuration from build-time to run-time, it will be regenerated on each restart, meaning that upgrading will set these in the Apache configuration:
    ```
    RewriteRule ^/?${MEDIAWIKI_WIKI_PREFIX:1}(/.*)?$ %{DOCUMENT_ROOT}/index.php
    RewriteRule ^/?${MEDIAWIKI_SCRIPT_PATH:1}(.*)$ %{DOCUMENT_ROOT}/\$1 [L]
    ```
- Because $MEDIAWIKI_SCRIPT_PATH is set to default `/w`, it will end up in the RewriteRule, even if MediaWiki was configured with `$wgScriptPath=""` when the server was first deployed.